roles/analytics-dashboard/tasks/custom.yml

```diff
@@ -15,3 +15,7 @@
 #----------------------------------------------------------------------------
 
 # Add configs for the custom files that are needed to be copied/replaced here.
+ - name: "Copy jar to bundle script"
```

Comment:
Keep the custom files empty. This is to be used by users to add their own customizations.

Reply:
fixed https://github.com/wso2/ansible-ei/pull/30/commits/bf084ed7e012838529727ccbb5adb33110a47243